docs/jwst/ami/main.rst

```diff
@@ -71,7 +71,12 @@ The ``ami_analyze`` step applies the Lacour-Greenbaum (LG) image plane
 modeling algorithm to a NIRISS AMI image.
 The routine computes a number of parameters, including a model fit (and
 residuals) to the image, fringe amplitudes and phases, and closure phases
-and amplitudes.
+and amplitudes.  For a full-frame exposure, instead of using the FULL subarray,
+a subarray will be extracted from its center to keep the execution time acceptable.
```

Comment:
Is the subarray actually at (or near) the center of the full image? If not, I would remove the words "from its center".